containers/mit-scheme/.devcontainer/library-scripts/common-debian.sh

```diff
@@ -5,15 +5,17 @@
 #-------------------------------------------------------------------------------------------------------------
 #
 # Docs: https://github.com/microsoft/vscode-dev-containers/blob/master/script-library/docs/common.md
+# Maintainer: The VS Code and Codespaces Teams
 #
-# Syntax: ./common-debian.sh [install zsh flag] [username] [user UID] [user GID] [upgrade packages flag] [install Oh My *! flag]
+# Syntax: ./common-debian.sh [install zsh flag] [username] [user UID] [user GID] [upgrade packages flag] [install Oh My Zsh! flag] [Add non-free packages]
 
 INSTALL_ZSH=${1:-"true"}
 USERNAME=${2:-"automatic"}
 USER_UID=${3:-"automatic"}
 USER_GID=${4:-"automatic"}
 UPGRADE_PACKAGES=${5:-"true"}
 INSTALL_OH_MYS=${6:-"true"}
+ADD_NON_FREE_PACKAGES=${7:-"false"}
```

Comment:
This was a recently merged PR that must have had an old version of the script in it. It would be auto-updated on release, so we should update it.  I just ran the `build/vscdc copy-library-scripts` command.

Reply:
Interestingly this was a recent PR that had the old version of the `common-debian.sh` script in it, so this should be updated.